CHANGES_NEXT_RELEASE

```diff
@@ -3,4 +3,5 @@
 - [cygnus][feature] Standardize logs layout (#1118)
 - [cygnus-common][hardening] Add implementation for /admin/log route (#807)
 - [cygnus-common][hardening] Create ManagementInterfaceUtils class (#1115)
+- [cygnus-common][bug] Add fiware-service and fiware-servicePath headers to Orion subcription method (#1124)
```

Comment:
In fact, the headers have been added to all the API operations, not only the subscriptions, right?